app/controllers/api/v1/lookup_keys_controller.rb

```diff
@@ -11,9 +11,10 @@ class LookupKeysController < V1::BaseController
       param :per_page, String, :desc => "number of entries per request"
 
       def index
-        @lookup_keys = LookupKey.
-          authorized(:view_external_variables).
-          search_for(*search_options).paginate(paginate_options)
+        @lookup_keys = PuppetclassLookupKey.authorized(:view_external_variables).
```

Comment:
what happens if the user passes an invalid search item for one of the lookup keys sub classes? imho it would break?